### PR TITLE
Remove aria-disabled attribute

### PR DESCRIPTION
**Pull request by pavi41**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.frontend/src/view/FormFieldBase.js [hunk 1]
```diff
@@ -450,14 +450,15 @@ class FormFieldBase extends FormField {
      * @param {Object} state - The state object.
      */
     updateEnabled(enabled, state) {
+        const screenReaderText = window.DOMPurify ? window.DOMPurify.sanitize(state.screenReaderText, { ALLOWED_TAGS: [] }) : state.screenReaderText;
```
Comment (rismehta) [inline ui.frontend/src/view/FormFieldBase.js:453]: Let's move this code out of FormFieldBase.js. You don't need to remove aria-label attribute here

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

File: ui.frontend/src/view/FormFieldBase.js (hunk 1 above)
Comment (pavi41) [reply ui.frontend/src/view/FormFieldBase.js]: Done!

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js [hunk 2]
```diff
@@ -74,6 +74,17 @@
             }
         }
 
+        updateEnabled(enabled, state) {
+            if (this.widget) {
+                this.element.setAttribute(FormView.Constants.DATA_ATTRIBUTE_ENABLED, enabled);
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js:79]: This should be included in FormFieldBase.js, as it's necessary for our runtime, such as setting the disable attribute and its related data attributes. Changes specific to HTML, like aria attributes, should belong to the view layer (i.e., datepickerview.js). You’ll also need to call super.updateEnabled for other functions, essentially invoking the method from FormFieldBase.

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

File: ui.frontend/src/view/FormFieldBase.js [hunk 3]
```diff
@@ -444,24 +441,6 @@ class FormFieldBase extends FormField {
         }
     }
 
-    /**
-     * Updates the HTML state based on the enabled state of the field.
-     * @param {boolean} enabled - The enabled state.
-     * @param {Object} state - The state object.
-     */
-    updateEnabled(enabled, state) {
```
Comment (rismehta) [inline ui.frontend/src/view/FormFieldBase.js:452]: You had to keep this function.

you just had to remove the following code from this,

```
this.widget.setAttribute(Constants.ARIA_DISABLED, true);
this.widget.removeAttribute(Constants.ARIA_DISABLED);
```

Comment (pavi41) [reply ui.frontend/src/view/FormFieldBase.js]: Done!

**Review by rismehta — APPROVED** [review]

**Pull request merged**